config/environments/test.rb

```diff
@@ -9,6 +9,9 @@
   # Settings specified here will take precedence over those in
   # config/application.rb.
 
+  config.en_host = "en.example.com"
```

Comment:
what's needed here?

Reply:
This is the test.rb, which runs in the test environment only,  so I just needed to provide some dummy values here, to stop the tests from failing (i.e. en.example.com is fine to keep as is, for test specs)